hnn_core/cell.py

```diff
@@ -696,6 +787,163 @@ def _update_end_pts(self):
 
         self._nrn_sections = dict()
 
+    def create_cell_tree(self):
+        """Create cell tree from topology
+
+        Returns
+        -------
+        Tree representation of the cell.
+        """
+        if self.topology is None:
+            return
+
+        end_pts_used = set()
+        # Make the soma entries
+        cell_tree = dict()
+
+        soma_0_end = ('soma', 0)  # end 0
+        soma_1_end = ('soma', 1)  # end 1
+
+        cell_tree[soma_0_end] = list()
+        # Connect end 1 to end 0
+        cell_tree[soma_0_end].append(soma_1_end)
+
+        end_pts_used.add(soma_0_end)
+        end_pts_used.add(soma_1_end)
+
+        # Add all connections
+        for connection in self.topology:
```

Comment:
if we got rid of `self.topology` would we need this function? how does the `self.cell_tree` look for a pyramidal cell? or should we still keep `self.topology`? They seem to be redundant ...